examples/entity/importexport/mysql-feed-import-direct-filesystem.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ A feed representing Hourly customer email data retained for 90 days
+ -->
+<feed description="Raw customer email feed" name="mysql-feed-import-direct-filesystem" xmlns="uri:falcon:feed:0.1">
+    <tags>externalSystem=USWestEmailServers,classification=secure</tags>
+    <groups>DataImportPipeline</groups>
+    <frequency>minutes(10)</frequency>
+    <late-arrival cut-off="hours(4)"/>
+    <clusters>
+        <cluster name="primaryCluster" type="source">
+            <validity start="2016-04-10T00:00Z" end="2017-03-31T00:00Z"/>
+            <retention limit="days(90)" action="delete"/>
+            <import> 
+                <source name="mysql-db" tableName="db_raw_data">
+                    <extract type="full">
+                        <mergepolicy>snapshot</mergepolicy>
+                    </extract>    
+                    <fields>
+                           <includes>
+                             <field>id</field>
+                             <field>name</field>
+                           </includes>
+                    </fields>
+                </source>
+                <arguments>
+                    <argument name="--split-by" value="id"/>
+                    <argument name="--num-mappers" value="2"/>
+                    <argument name="--direct" value="true"/>
+                    <argument name="--direct.default-character-set" value="latin1"/>
```

Comment:
Let us have a single extraArgs as a string list
For example
   <argument name="ConnectorExtraArg"  value="--charset=utf8 --schema=abc"/>